### PR TITLE
Many improvements, corrections, and simplifications

### PR DESCRIPTION
**Pull request by Gitoffthelawn**
One of the biggest improvements is making the links to `dynamic-theme-fixes.config` and `inversion-fixes.config` easier to find at quick glance.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,66 +1,67 @@
...
 ## Sponsor
 
-<a href="https://opencollective.com/darkreader/donate" target="_blank" rel="noreferrer noopener"> <img src="https://opencollective.com/darkreader/donate/button@2x.png?color=blue" width=300 /></a>
+Sponsor the development of Dark Reader by clicking here:
 
-Sponsor the development of Dark Reader
+<a href="https://opencollective.com/darkreader/donate" target="_blank" rel="noreferrer noopener"> <img src="https://opencollective.com/darkreader/donate/button@2x.png?color=blue" width=300 /></a>
 
-## Translation
+## Translate
```
Comment (Gusted) [inline CONTRIBUTING.md:12]: I think `Translation` is better as `Translate` is a verb and should be used in a sentence-like way.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,66 +1,67 @@
...
 
 [Improve or suggest](https://github.com/darkreader/darkreader/tree/master/src/_locales) a translation.
-See the list of [language codes](https://developer.chrome.com/webstore/i18n#localeTable) that we can support.
+See the list of [language codes](https://developer.chrome.com/webstore/i18n#localeTable) we can support.
 
-## Adding a website that is already dark
+## Indicate a Website is Already Dark
```
Comment (Gusted) [inline CONTRIBUTING.md:17]:
```suggestion
## Indicate a website is already dark
```

No unnecessary capital letters.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,66 +1,67 @@
...
+- The URL is the actual website address (no CNAME or URL redirects).
+- The page isn't WIP (sites will change in this phase and could afterwards not meet the requirements).
 
-Then, you can **add it to the [dark-sites.config](https://github.com/darkreader/darkreader/blob/master/src/config/dark-sites.config) file**
-*(please, preserve the alphabetical order)*.
+Then, you can add it to this file *(please preserve alphabetic order)*:  
+- **[dark-sites.config](https://github.com/darkreader/darkreader/blob/master/src/config/dark-sites.config)**
 
-## Fixing incorrect inversions
+## Fix Incorrect Inversions
```
Comment (Gusted) [inline CONTRIBUTING.md:27]:
```suggestion
## Fix incorrect inversions
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,66 +1,67 @@
...
 
 If certain **parts** of a web page are **incorrectly inverted or styled**,
-you can fix this by specifying the appropriate [**CSS selectors**](https://developer.mozilla.org/docs/Web/CSS/CSS_Selectors) in
-**[dynamic-theme-fixes.config](https://github.com/darkreader/darkreader/blob/master/src/config/dynamic-theme-fixes.config)**
-(for Dynamic Theme mode)
-or **[inversion-fixes.config](https://github.com/darkreader/darkreader/blob/master/src/config/inversion-fixes.config)**
-(for Filter and Filter+ modes) *(__Please, preserve alphabetical order by URL, use short selectors, and preserve code style__)*.
+you can fix this by specifying the appropriate [**CSS selectors**](https://developer.mozilla.org/docs/Web/CSS/CSS_Selectors)
+in the appropriate file *(please preserve alphabetic order by URL, use short selectors, and preserve code style)*:
```
Comment (Gusted) [inline CONTRIBUTING.md:31]:
```suggestion
in the appropriate file *(please preserve alphabetic order by URL, and preserve code style)*:  
```
Shorter isn't always better.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,66 +1,67 @@
...
-(for Filter and Filter+ modes) *(__Please, preserve alphabetical order by URL, use short selectors, and preserve code style__)*.
+you can fix this by specifying the appropriate [**CSS selectors**](https://developer.mozilla.org/docs/Web/CSS/CSS_Selectors)
+in the appropriate file *(please preserve alphabetic order by URL, use short selectors, and preserve code style)*:  
+- Fixes for Dynamic Theme mode: **[dynamic-theme-fixes.config](https://github.com/darkreader/darkreader/blob/master/src/config/dynamic-theme-fixes.config)**  
+- Fixes for Filter/Filter+ modes: **[inversion-fixes.config](https://github.com/darkreader/darkreader/blob/master/src/config/inversion-fixes.config)**
 
 Below you can learn how to make a fix for the appropriate mode.
 
-## How to use the Dev Tools
+## How to Use Dev Tools
```
Comment (Gusted) [inline CONTRIBUTING.md:37]: Revert this. I don't think this is correct.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,66 +1,67 @@
...
 - Click **Edit** (login to GitHub first).
-- **Insert your fix** there. Preserve **alphabetic order** by URL.
+- Preserving *alphabetic order by URL*, **insert your fix** there.
 - Provide a **short description** of what you have done.
 - Click **Propose file change**.
 - Review your changes. Click **Create pull request**.
-- Github actions will run tests to make sure it has the right code-style.
-- If you see a **red cross** click **Details** to see what is wrong and edit the existing Pull Request.
-- When you see a **green checkmark** then everything is fine.
+- Github actions will run tests to make sure youe changes have the correct code style.
```
Comment (Gusted) [inline CONTRIBUTING.md:62]:
```suggestion
- Github actions will run tests to make sure your changes have the correct code style.
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -92,11 +93,11 @@ IGNORE IMAGE ANALYSIS
 | **IGNORE&nbsp;INLINE&nbsp;STYLE** | Prevents inline style analysis of matched elements. | *Example*: `<p style="color: red">` element's style attribute will not be changed. |
 | **IGNORE&nbsp;IMAGE&nbsp;ANALYSIS** | Prevents background images from being analyzed for matched selectors. |  |
 
-## Adding a new color scheme
+## Add a New Color Scheme
```
Comment (Gusted) [inline CONTRIBUTING.md:96]:
```suggestion
## Add a new color scheme
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -190,17 +192,17 @@ Then execute `npm run debug`.
...
 ## Tips
 
-For editing the code you can use any text editor or web IDE (like [Visual Studio Code](https://code.visualstudio.com), [Atom](https://atom.io/), or [WebStorm](https://www.jetbrains.com/webstorm/)).
+For editing the code, you can use any text editor or web IDE (like [Visual Studio Code](https://code.visualstudio.com), [Atom](https://atom.io/), or [WebStorm](https://www.jetbrains.com/webstorm/)).
 
 **Please preserve code style** (whitespaces etc).
 This can automatically be done by executing `npm run code-style`.
 
-Run tests by executing `npm test` to make sure it will pass the test.
+Run tests by executing `npm test` to make sure your code will pass.
```
Comment (Gusted) [inline CONTRIBUTING.md:205]: Pass what? 😄

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -106,11 +107,11 @@ Please follow the next steps to add a new color scheme:
...
 - Github actions will run tests to make sure it has the right code-style.
-- If you see a **red cross** click **Details** to see what is wrong and edit the existing Pull Request.
-- When you see a **green checkmark** then everything is fine.
-- Dark Reader developer will **review** and merge your changes, making them available.
+- If you see a **red cross** (`x`), click **Details** to see what is wrong, and edit the existing *Pull Request*.
+- When you see a **green checkmark**, everything is fine.
+- Dark Reader developer will **review** and merge your changes, making them available to everyone.
 
-## Dynamic variables
+## Dynamic Variables
```
Comment (Gusted) [inline CONTRIBUTING.md:114]:
```suggestion
## Dynamic variables
```

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -131,15 +132,15 @@ CSS
...
+Here are all the available CSS variables:
 | Variable | Description | Use |
 |---|---|---|
 | **`--darkreader-neutral-background`** | Neutral background color that <br>corresponds to the user's settings. | Mostly used for elements that have <br>a wrong background color |
 | **`--darkreader-neutral-text`** | Neutral text color that <br>corresponds to the user's settings. | Used for elements with a wrong text color |
 | **`--darkreader-selection-background`** | The text color setting <br>defined by the user. | The user's Text Color setting |
 | **`--darkreader-selection-text`** | The background color setting <br>defined by the user. | The user's Background Color setting |
 
-## Fixes for Filter and Filter+ mode
+## Fixes for Filter & Filter+ modes
```
Comment (Gusted) [inline CONTRIBUTING.md:143]:
```suggestion
## Fixes for Filter & Filter+ mode
```

Modes implies that there are more a-like modes.

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -164,19 +165,20 @@ CSS
 }
 ```
 
-- Filter and Filter+ work by inverting the whole web page and reverting necessary parts (images, videos, etc.), listed in the `INVERT` section.
-- If an inverted element contains images or other content that becomes incorrectly displayed, `NO INVERT` rule can be used.
+- Filter and Filter+ modes work by inverting the whole web page and reverting necessary elements (images, videos, etc.) listed in the `INVERT` section.
```
Comment (Gusted) [inline CONTRIBUTING.md:168]:
```suggestion
- Filter and Filter+ mode work by inverting the whole web page and reverting necessary elements (images, videos, etc.) listed in the `INVERT` section.
```

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -164,19 +165,20 @@ CSS
...
+- If an inverted element contains images or other content that becomes incorrectly displayed, the `NO INVERT` rule can be used.
 - `REMOVE BG` removes the background image from an element and forces a black background.
 
-## Adding new features or fixing bugs
+## Add New Features or Fix Bugs
```
Comment (Gusted) [inline CONTRIBUTING.md:172]:
```suggestion
## Add new features or fix bugs
```

**Review by Gusted — COMMENTED** [review]
A few thingies, but overall good.

File: CONTRIBUTING.md (hunk 1 above)
Comment (Gitoffthelawn) [reply CONTRIBUTING.md:12]: Either is good by me.  I wanted to encourage action, so I used verbs.  Also, the first item is "Sponsor", which is being used as a verb, so I wanted to be consistent for all the headings.

Comment (Gusted) [reply CONTRIBUTING.md:12]: Well then you can change `Sponsor` to `Sponsoring` or something alike.

File: CONTRIBUTING.md (hunk 8 above)
Comment (Gitoffthelawn) [reply CONTRIBUTING.md:205]: The *tests* (second word).  I can add "them" to the end of the sentence if you like.  What's the best way to edit what I've already issued a PR for?  When you make a suggestion, I can just accept it quickly (if I like it 😸 ), but I'm not seeing a quick way for me to just add a word to the PR without finding the file in my fork.

File: CONTRIBUTING.md (hunk 10 above)
Comment (Gitoffthelawn) [reply CONTRIBUTING.md:143]: Aren't Filter and Filter+ separate modes?  They are in the UI.

File: CONTRIBUTING.md (hunk 8 above)
Comment (Gusted) [reply CONTRIBUTING.md:205]:
```suggestion
Run tests by executing `npm test` to make sure your code will pass them.
```

Comment (Gusted) [reply CONTRIBUTING.md:205]: > What's the best way to edit what I've already issued a PR for?

File changed tab -> 3 dots on the first file -> Edit file.

File: CONTRIBUTING.md (hunk 10 above)
Comment (Gusted) [reply CONTRIBUTING.md:143]: They are, but if you both mention them you don't need to include the `s`, if you only mention `Filter` then you can use `modes`. At least, that's how I remember my English classes, and it does read better that way.

File: CONTRIBUTING.md (hunk 1 above)
Comment (Gitoffthelawn) [reply CONTRIBUTING.md:12]: You *really* don't like verbs?

File: CONTRIBUTING.md [hunk 13]
```diff
@@ -1,66 +1,67 @@
...
 
-Then, you can **add it to the [dark-sites.config](https://github.com/darkreader/darkreader/blob/master/src/config/dark-sites.config) file**
-*(please, preserve the alphabetical order)*.
+Then, you can add it to this file *(please preserve alphabetic order)*:  
+- **[dark-sites.config](https://github.com/darkreader/darkreader/blob/master/src/config/dark-sites.config)**
 
-## Fixing incorrect inversions
+## Fix incorrect inversions
 
 If certain **parts** of a web page are **incorrectly inverted or styled**,
```
Comment (q1800) [inline CONTRIBUTING.md:29]:
```suggestion
If certain **elements** of a web page are **incorrectly inverted or styled**,
```

File: CONTRIBUTING.md [hunk 14]
```diff
@@ -1,66 +1,67 @@
...
 Below you can learn how to make a fix for the appropriate mode.
 
-## How to use the Dev Tools
+## How to Use Dev Tools
 
 Dev Tools is designed to **fix minor issues** on a web page
-*(like dark icon on a dark background, removing a bright background, adding a white background to a transparent image, etc.)*.
+*(like a dark icon on a dark background, removing a bright background, adding a white background to a transparent image, etc.)*.
 If the page looks partially dark and bright in **Dynamic mode**, it's considered a bug.
 For **Filter mode**, it is a common practice to invert parts of the page that are already dark.
```
Comment (q1800) [inline CONTRIBUTING.md:42]:
```suggestion
For **Filter mode**, it is a common practice to invert elements of the page that are already dark.
```
How about we change the word parts here to elements to help with more consistency in the document? Just like the change that was made on Line 168.

File: CONTRIBUTING.md (hunk 13 above)
Comment (Gitoffthelawn) [reply CONTRIBUTING.md:29]: Great idea!

File: CONTRIBUTING.md (hunk 14 above)
Comment (Gusted) [reply CONTRIBUTING.md:42]: > Is this PR able to be merged? Are we waiting on anything else here?

See open suggested changes.

Comment (q1800) [reply CONTRIBUTING.md:42]: Oh alright, I understand now. I didn't know if @Gitoffthelawn had seen all of these open suggestions still pending. Just curious, not trying to be a pest or pushy sounding.

Thank you again. 😃

Comment (Gitoffthelawn) [reply CONTRIBUTING.md:42]: Hi everyone.  Just very busy.  I'll definitely get back to this as time permits.

**Pull request closed without merging**